### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: caseworker/ui_automation_tests/fixtures/add_queue.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from pytest import fixture
-import shared.tools.helpers as utils
+import tests_common.tools.helpers as utils
 from pages.queues_pages import QueuesPages
-from pages.shared import Shared
+from pages.shared import tests_common
```
Comment (r4vi) [inline caseworker/ui_automation_tests/fixtures/add_queue.py:4]: can this be imported directly?

File: caseworker/ui_automation_tests/pages/generate_document_page.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
-from pages.shared import Shared
+from pages.shared import tests_common
```
Comment (r4vi) [inline caseworker/ui_automation_tests/pages/generate_document_page.py:1]: I can't see this being used

File: caseworker/ui_automation_tests/pages/letter_templates.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 from pages.BasePage import BasePage
 
-from caseworker.ui_automation_tests.pages.shared import Shared
+from caseworker.ui_automation_tests.pages.shared import tests_common
```
Comment (r4vi) [inline caseworker/ui_automation_tests/pages/letter_templates.py:3]: keep seeing this and I think it's a find/replace mistake. This should not really change to `tests_common` there is a class in `pages.shared` called `Shared`

File: caseworker/ui_automation_tests/fixtures/add_queue.py (hunk 1 above)
Comment (richtier) [reply caseworker/ui_automation_tests/fixtures/add_queue.py:4]: oh 100%. result of find and replace :+1:

File: caseworker/ui_automation_tests/pages/letter_templates.py (hunk 3 above)
Comment (richtier) [reply caseworker/ui_automation_tests/pages/letter_templates.py:3]: agreed. will fix :+1:

File: caseworker/ui_automation_tests/fixtures/add_queue.py [hunk 4]
```diff
@@ -10,4 +10,4 @@ def add_queue(driver, request, api_url, context):
     extra_string = str(utils.get_formatted_date_time_d_h_m_s())
     context.queue_name = "Review" + extra_string
     QueuesPages(driver).enter_queue_name(context.queue_name)
-    Shared(driver).click_submit()
+    tests_common.Shared(driver).click_submit()
```
Comment (r4vi) [inline caseworker/ui_automation_tests/fixtures/add_queue.py:13]:
```suggestion
    Shared(driver).click_submit()
```

**Pull request merged**